image-builder/image_builder.sh

```diff
@@ -382,6 +382,9 @@ create_rootfs_image() {
 		find "${mount_dir}" -type f -name "${u}" -exec rm -f {} \;
 	done
 
+	info "Creating empty machine-id to allow systemd bind-mount it"
```

Comment:
Nit: s/systemd bind-mount/systemd to bind-mount/

Reply:
fixed, thanks